resources/styles/main.scss

```diff
@@ -7,7 +7,8 @@
 @use "atoms/breadcrumbs";
 @use "atoms/button";
 @use "atoms/dropdown";
-//@use "atoms/field/field.scss";
+@use "atoms/field/field.scss";
```

Comment:
let's rename field.scss and select.scss to _index.scss in their respective directories so we can
```
@use "atoms/field";
@use "atoms/select";
```